tests/view_tests/tests/test_static.py

```diff
@@ -124,6 +126,20 @@ def tearDown(self):
         super().tearDown()
         urls.urlpatterns = self._old_views_urlpatterns
 
+    def test_normal_result(self):
+        self.assertEqual(str(static('test')[0]), str(url('^test(?P<path>.*)$', serve)))
+
+    @override_settings(DEBUG=False)
+    def test_debug_off(self):
+        self.assertEqual(static('test'), [])
```

Comment:
This test and `test_special_prefix` might not be self-evident so a docstring wouldn't hurt, e.g.
`"""No URLs are served if DEBUG=False."""`